package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "template-instance",
-  "version": "0.1.0",
+  "version": "0.7.2",
```

Comment:
This is the template-instance version, which is independent from the sourcecred version. No reason to change it.